tensorboard/webapp/metrics/views/_common.scss

```diff
@@ -61,6 +61,10 @@ $metrics-min-card-height: 320px;
 @mixin metrics-card-controls {
   @include tb-theme-foreground-prop(color, secondary-text);
   white-space: nowrap;
+
+  ::ng-deep button[mat-icon-button] {
+    font-size: 16px;
```

Comment:
This is unnecessary with my other suggestions. I think the 24px x 24px default is perfectly fine here.